py/load_firmware.py

```diff
@@ -203,7 +203,7 @@ def main() -> int:
         return 1
 
     if args.usart is not None:
-        serial_port = usart.SerialPort(args.usart, 115200, timeout=3)
+        serial_port = usart.SerialPort(args.usart, timeout=3)
```

Comment:
For the same reason as above, this stray `timeout` argument should be removed (as it's going to be ignored)

Reply:
fixed